region/brinstar/green/Etecoon Energy Tank Room.json

```diff
@@ -323,16 +333,17 @@
       "link": [1, 1],
       "name": "Leave Space Jumping",
       "requires": [
-        "canPreciseSpaceJump"
+        "canTrickyJump"
```

Comment:
Should this and `Leave Spinning` requirements be swapped?

Reply:
The "SpaceJump" item requirement is implicit in the exit condition "leaveSpaceJumping" so it's not needed here. Whereas for the "leaveSpinning" strat, we have to include it explicitly. 

The "Leave Spinning" runway length didn't look right though, so I just updated it: when going for max speed (rather than shortcharging) you have to use a little less than full runway in order to be able to Space Jump early enough to control your position through the door as this exit condition requires.